src/unistd.rs

```diff
@@ -737,11 +737,25 @@ pub fn write(fd: RawFd, buf: &[u8]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
+/// Directive that tells [`lseek`] and [`lseek64`] how to reposition the offset of the file.
```

Comment:
"Directive for [`lseek`] and [`lseek64`] that specifies what the `offset` is relative to."